addons/arsenal/functions/fnc_onSelChangedLeft.sqf

```diff
@@ -56,7 +56,7 @@ switch (GVAR(currentLeftPanel)) do {
             if ((GVAR(currentItems) select 0) != _item && {_baseWeapon != _item}) then {
                 call _fnc_clearPreviousWepMags;
 
-                private _compatibleItems = (_item call bis_fnc_compatibleItems) apply {tolower _x};
+                private _compatibleItems = _item call bis_fnc_compatibleItems;
```

Comment:
```suggestion
                private _compatibleItems = _item call BIS_fnc_compatibleItems;
```

you cannot rely on BIS_fnc_compatibleItems returning config case, should sanitize this input